browser/model/helpers/installer.js

```diff
@@ -0,0 +1,120 @@
+'use strict';
+
+let fs = require('fs-extra');
+let child_process = require('child_process');
+let unzip = require('unzip');
+
+import Logger from '../../services/logger';
+
+class Installer {
+
+  constructor(key, progress, success, failure) {
+    this.progress = progress;
+    this.success = success;
+    this.failure = failure;
+    this.key = key;
+  }
+
+  exec(command, options, result) {
+    let self = this;
+    return new Promise(function (resolve, reject) {
+      Logger.info(self.key + ' - Execute ' + command);
+      child_process.exec(command, options, function(error, stdout, stderr) {
```

Comment:
True, but the reason for it to be there is because it is wrapped in a promise like this:
```javascript
return new Promise(function (resolve, reject) {
```
Using arrow functions won't really affect anything right now, because I need the self variable for logging before calling the actual function.